app/controllers/applications_controller.rb

```diff
@@ -4,8 +4,14 @@ def new
   end
 
   def create
-    application = Application.create(app_params)
-    redirect_to "/applications/#{application.id}"
+    application = Application.new(app_params)
+    if application.save
+      redirect_to "/applications/#{application.id}"
+      flash[:alert] = "Application Started Successfully!"
+    else
+      redirect_to "/applications/new"
+      flash[:alert] = "Error: #{error_message(application.errors)}"
```

Comment:
i was wondering if we could use this functionality that is already built-in https://github.com/westonio/adopt-dont-shop-7/blob/5f5bf3fe4b1f37324cf12dd8b1ac1150a33ea4e4/app/controllers/application_controller.rb#L7 but this is totally fine, too! just saw this and thought maybe we could use it.